dashboard/app/helpers/levels_helper.rb

```diff
@@ -770,4 +770,19 @@ def can_view_teacher_markdown?
   def include_multi_answers?(standalone)
     standalone || current_user.try(:should_see_inline_answer?, @script_level)
   end
+
+  def get_level_source_image(level_image, level_source_id)
```

Comment:
Please document the function. Is `level_image` a string? That `level_source_id` is an integer or nil. It returns a `LevelSourceImage` or nil. Etc.